styles/general.css

```diff
@@ -1,4 +1,13 @@
 body{
     margin: 0;
     font-family: 'IM Fell DW Pica', serif;
-}
\ No newline at end of file
+}body{
+    overflow-x: hidden;
+    margin: 0;
+    margin-top: 70px;
+    font-family: 'IM Fell DW Pica', serif;
+}
+:root {
+    --green-background: #44CC66;
```

Comment:
1. please, don't just copy the code from lectures, you have to put your own effort
2. there is no variables for different themes and variables with pixels too